### PR TITLE
Nataliia romanchuk/exceptions

### PR DESCRIPTION
**Pull request by Vivanatalya**
Tested different types of exceptions

### CODE REVIEW

File: tests/nataliia_romanchuk/test_exceptions.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+import pytest
+
+from nataliia_romanchuk.Classes import Cats
+
+
+def test_attribute_error():
+    cat = Cats('Pushok', 10)
+    print(cat.meow())
+    print(cat.sleep())
```
Comment (kudinov-fedor) [inline tests/nataliia_romanchuk/test_exceptions.py:11]: avoid prints since they require visual assertion, 
change it to 
assert

**Review by kudinov-fedor — APPROVED** [review]

**Review by kudinov-fedor — APPROVED** [review]

File: tests/nataliia_romanchuk/test_exceptions.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    with pytest.raises(ArithmeticError):
+        1 / 0
+
+
+def test_file_not_found_error():
+    package_name = "nataliia_romanchuk"
+    with pytest.raises(FileNotFoundError):
+        with importlib.resources.open_text(package_name, "Teext.txt") as file:
+            content = file.read()
+            print(content)
```
Comment (kudinov-fedor) [inline tests/nataliia_romanchuk/test_exceptions.py:42]: minimum code required


```
def test_file_not_found_error():
    
    with pytest.raises(FileNotFoundError):
        importlib.resources.open_text("nataliia_romanchuk", "Teext.txt") 
```

File: tests/nataliia_romanchuk/test_exceptions.py [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+
+def test_module_not_found_error():
+    package_name = "nataliia_romanchukkkkkkk"
+    with pytest.raises(ModuleNotFoundError):
+        with importlib.resources.open_text(package_name, "Text.txt") as file:
+            content = file.read()
+            print(content)
```
Comment (kudinov-fedor) [inline tests/nataliia_romanchuk/test_exceptions.py:50]: same here

File: tests/nataliia_romanchuk/test_exceptions.py [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+
+def test_value_error():
+    with pytest.raises(ValueError):
+        int('qwerty')
+
+
+def test_memory_error():
+    with pytest.raises(MemoryError):
+        bytearray(100 ** 9)
```
Comment (kudinov-fedor) [inline tests/nataliia_romanchuk/test_exceptions.py:65]: oh wow ))

File: nataliia_romanchuk/Classes.py [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+    def __init__(self, name, age):
+        self.name = name
+        self.age = age
+
+    def meow(self):
+        return f"Cat {self.name} says Meow for " + f"{self.age} years!"
+
+    def sleep(self):
+        return f"Cat {self.name} is sleeping. Shhhhhhhh!"
+
```
[the remark this answers is not in the document]
Comment (Vivanatalya) [reply nataliia_romanchuk/Classes.py:11]: done

File: tests/nataliia_romanchuk/test_exceptions.py (hunk 1 above)
Comment (Vivanatalya) [reply tests/nataliia_romanchuk/test_exceptions.py:11]: fixed

File: tests/nataliia_romanchuk/test_exceptions.py (hunk 2 above)
Comment (Vivanatalya) [reply tests/nataliia_romanchuk/test_exceptions.py:45]: fixed

File: tests/nataliia_romanchuk/test_exceptions.py (hunk 3 above)
Comment (Vivanatalya) [reply tests/nataliia_romanchuk/test_exceptions.py:53]: fixed

**Pull request merged**